README.md

```diff
@@ -1,18 +1,176 @@
 # Community
 
-## Become a member of rust-vmm
+## What is rust-vmm?
 
-1. Submit a new [access request issue](https://github.com/rust-vmm/community/issues/new?assignees=andreeaflorescu%2C+jbryce%2C+sameo&labels=&template=access-request-to-rust-vmm-organization.md&title=Access+Request+to+rust-vmm).
+rust-vmm is an open-source project empowering the community to build custom
+Virtual Machine Monitors (VMMs) and Hypervisors. It provides a set of
+virtualization components that any project can use to quickly develop
+virtualization solutions while focusing on the key differentiators of their
+product rather than re-implementing common components like KVM wrappers, virtio
+devices and other VMM libraries.
```

Comment:
It's definitely not a blocker for a first version. I suggest you get more wring feedback once this document starts to get more user traffic, since at that point clarity will matter more.